rotkehlchen/rotkehlchen.py

```diff
@@ -910,6 +916,14 @@ def query_balances(
             save_despite_errors=save_despite_errors,
         )
 
+        if detect_tokens is not None:
+            for blockchain, addresses in detect_tokens.items():
+                manager: EvmManager = self.chains_aggregator.get_chain_manager(blockchain)
+                try:
+                    manager.tokens.detect_tokens(only_cache=False, addresses=addresses)
+                except (RemoteError, BadFunctionCallOutput) as e:
+                    log.error(f'Failed to detect tokens: {e}')
```

Comment:
can you provide here the chain and the addresses quried? ALso mention that the detection for that chain is skipped